crates/test_utils/src/main.rs

```diff
@@ -0,0 +1,163 @@
+use std::{
+    env,
+    process::{exit, Command as cmd},
+};
+
+use clap::{arg, command, Parser};
+use router::types::ConnectorAuthType;
+use test_utils::connector_auth::ConnectorAuthenticationMap;
+
+// Just by the name of the connector, this function generates the name of the collection
+// Example: CONNECTOR_NAME="stripe" -> OUTPUT: postman/stripe.postman_collection.json
+fn path_generation(name: &String) -> String {
+    format!("postman/{}.postman_collection.json", name)
+}
+
+#[derive(Parser)]
+#[command(author, version, about = "Postman collection runner using newman!", long_about = None)]
```

Comment:
I don't think we've specified authors in `Cargo.toml`. You can remove it from here.